website/docs/gator.md

```diff
@@ -240,6 +240,24 @@ the `run` flag:
 gator verify path/to/suites/... --run "disallowed"
 ```
 
+#### Validating Metadata-Based Constraint Templates
+
+`gator verify` may be used with an [`AdmissionReview`](https://pkg.go.dev/k8s.io/kubernetes/pkg/apis/admission#AdmissionReview) 
+object to test your constraints. This can be helpful to simulate a certain operation (`CREATE`, `UPDATE`, `DELETE`, etc.) 
+or [`UserInfo`](https://pkg.go.dev/k8s.io/kubernetes@v1.25.3/pkg/apis/authentication#UserInfo) metadata. 
+Recall that the `input.review.user` can be accessed in the Rego code (see [Input Review](howto.md#input-review) for more guidance). 
+A few examples for how to structure your yaml can be found [here](https://github.com/open-policy-agent/gatekeeper/blob/master/pkg/gator/fixtures/fixtures.go#L506-L528). 
```

Comment:
Use permalink instead in case the code moves around off the master branch.
```suggestion
A few examples for how to structure your yaml can be found [here](https://github.com/open-policy-agent/gatekeeper/blob/03e6adb74f1714242cf936fd27eee19a0eda2d52/pkg/gator/fixtures/fixtures.go#L506-L528). 
```